altair/utils/mimebundle.py

```diff
@@ -43,16 +45,16 @@ def spec_to_mimebundle(
 
     Note
     ----
-    The png, svg, pdf, and vega outputs require the altair_saver package
+    The png, svg, vega and pdf outputs require the vl-convert or altair_saver package
     to be installed.
     """
-    if mode not in ["vega", "vega-lite"]:
-        raise ValueError("mode must be either 'vega' or 'vega-lite'")
+    if mode != "vega-lite":
+        if mode == "vega":
+            warnings.warn(
+                "mode 'vega' is deprecated, use 'vega-lite'", AltairDeprecationWarning
```

Comment:
I like that this informs users of the change. Should it raise a value error within the if statement with this message instaed of a deprecation warning? I usually encounter deprecation warnings as "something will be deprecated in a **future** release but right now it still works so this does not throw an exception", not "something is already removed".